_pkgdown.yaml

```diff
@@ -26,6 +26,10 @@ reference:
     contents:
       - ends_with("linter")
 
+  - title: Grouped linters
```

Comment:
I am not sure about this title. But I also can't think of a better one.

Reply:
"Groups of linters"?